cypress/e2e/home_page.cy.ts

```diff
@@ -0,0 +1,21 @@
+const viewPorts = [
+  { name: 'iphone-8', dimensions: [375, 667] },
+  { name: 'desktop', dimensions: [1920, 1080] },
+];
```

Comment:
For now I'll keep 'dimensions' key instead of suggested 'preset' since I don't see preset with 1920x1080 px in cypress docs - https://docs.cypress.io/api/commands/viewport